packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -93,15 +96,18 @@ export class RunningSummarizer implements IDisposable {
         firstAck: ISummaryAttempt,
```

Comment:
What is `firstAck` here? If this is the first time we are creating a running summarizer, how can there already an ack?
Or is this just storing information about the first summary attempt?

Reply:
Renamed to `attemptBaseline` so it's more clear that it's just the baseline attempt data, not necessarily an actual summary attempt.